minorminer/utils/parallel_embeddings.py

```diff
@@ -0,0 +1,630 @@
+        if tile is None:
+            tile = dnx.chimera_graph(m=sublattice_size, n=sublattice_size, t=t)
+        elif (
+            use_filter
+            and embedding_feasibility_filter(S, tile, not one_to_iterable) is False
+        ):
+            warnings.warn("tile is infeasible: embeddings will be empty.")
+            return []
+    elif family == "pegasus":
+        sublattice_mappings = dnx.pegasus_sublattice_mappings
+        if tile is None:
+            tile = dnx.pegasus_graph(m=sublattice_size)
+        elif (
+            use_filter
+            and embedding_feasibility_filter(S, tile, not one_to_iterable) is False
+        ):
+            warnings.warn("tile is infeasible: embeddings will be empty.")
+            return []
+    elif family == "zephyr":
+        sublattice_mappings = dnx.zephyr_sublattice_mappings
+        t = T.graph["tile"]
+        if tile is None:
+            tile = dnx.zephyr_graph(m=sublattice_size, t=t)
+        elif (
+            use_filter
+            and embedding_feasibility_filter(S, tile, not one_to_iterable) is False
+        ):
+            warnings.warn("tile is infeasible: embeddings will be empty.")
+            return []
+    else:
+        raise ValueError(
+            "source graphs must a graph constructed by "
+            "dwave_networkx as chimera, pegasus or zephyr type"
+        )
+
+    tiling = tile == S
+    embs = []
+    if max_num_emb == 1 and seed is None:
+        _T = T
+    else:
+        _T = T.copy()
+
+    if shuffle_all_graphs or shuffle_sublattice_order:
+        prng = np.random.default_rng(seed)
+    if shuffle_sublattice_order:
+        sublattice_iter = list(sublattice_mappings(tile, _T))
+        prng.shuffle(sublattice_iter)
+    else:
+        sublattice_iter = sublattice_mappings(tile, _T)
+
+    for i, f in enumerate(sublattice_iter):
+        Tr = _T.subgraph([f(n) for n in tile])
+        if tiling:
+            if Tr.number_of_edges() == S.number_of_edges():
+                sub_embs = [{k: v for k, v in zip(S.nodes, Tr.nodes)}]
+            else:
+                sub_embs = []
+        else:
+            sub_embs = find_multiple_embeddings(
+                S,
+                Tr,
+                max_num_emb=max_num_emb,
+                use_filter=use_filter,
+                seed=seed,
+                embedder=embedder,
+                embedder_kwargs=embedder_kwargs,
+            )
+        embs += sub_embs
+        if len(embs) >= max_num_emb:
+            break
+
+        for emb in sub_embs:
+            # A potential feature extension would be to generate many
+            # overlapping embeddings and solve an independent set problem. This
+            # may allow additional parallel embeddings.
+            if one_to_iterable:
+                _T.remove_nodes_from([v for c in emb.values() for v in c])
+            else:
+                _T.remove_nodes_from(emb.values())
+
+    return embs
+
+
+if __name__ == "__main__":
```

Comment:
This file should not contain a `__main__` section. As @boothby mentioned, you could move this over to the examples, adding a new example for using the paralellel embeddings utils. I'm fine with keeping the above functions here though, apart from perhaps `visualize_embeddings` which seems like it would fit better with the other graph visualization functions in `dwave-networkx`.